go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/minio/minio-go/v7 v7.0.49
 	github.com/mwitkow/go-proto-validators v0.3.2
 	github.com/percona-platform/dbaas-api v0.0.0-20221214134301-b6a331250826
-	github.com/percona-platform/saas v0.0.0-20221014123257-4fa7a15ce672
+	github.com/percona-platform/saas v0.0.0-20230224070200-7fc92a89ba43
```

Comment:
todo(@idoqo ) revert when [percona-platform#68](https://github.com/percona-platform/saas/pull/68) is merged.